protocols/virtualfund/virtualfund.go

```diff
@@ -109,7 +171,9 @@ func constructFromState(
 	initialStateOfV state.State,
 	myAddress types.Address,
 	ledgerChannelToMyLeft *channel.TwoPartyLedger,
+	consensusChannelToMyLeft *consensus_channel.ConsensusChannel,
 	ledgerChannelToMyRight *channel.TwoPartyLedger,
+	consensusChannelToMyRight *consensus_channel.ConsensusChannel,
```

Comment:
aside: This PR would have been a little cleaner if this weren't added to the `constructFromState` signature -- `nil` is the only value that's passed in this PR.

Reply:
This is approximately "as intended"

1. add no-op side-by-side entry points
2. side-by-side implementations
3. remove existing implementation

... but you correctly assert that this PR is messy. "Getting it in" should make a path for cleaner future work.